_specs/ecip-1100.md

```diff
@@ -0,0 +1,207 @@
+
+![mess-acceptreject](https://user-images.githubusercontent.com/45600330/92998215-2fcb6f80-f4de-11ea-8af9-178d905c9086.png)
+
+## Implementation
+
+This feature does not require a hard fork, but the network stands to benefit and avoid risk with majority coordinated acivation.
+
+__Core-Geth__
+
+- Feature is tentatively proposed to activate in Core-Geth on Ethereum Classic network at and above block 11295300.
+- Feature is proposed to activate in Core-Geth on Ethereum Classic's Mordor test network at and above block 2290740.
+
+- Core-Geth feature implementation includes a few additional safety mechanisms:
+  + MESS is disabled for any sync mode besides full sync.
+  + MESS is only enabled once a peer has completed initial chain synchronisation, not while they are fast syncing or even full syncing during the download and process phase. This reduces the chances of a node coming online being lured into an eclipse scenario.
+  + MESS is only enabled if a peer has greater than or equal to the `2 * MinimumSyncPeers` peers. In Core-Geth this value is by default `5`, yielding a necessary limit of `10` for the MESS feature.
+  + MESS is disabled if, once synced, a node's head block is not changed within a time limit (ie becomes stale).  In Core-Geth this value is by default `10 * 13 seconds`.
+  
+The associated Core-Geth implementation is available [here](https://github.com/etclabscore/core-geth/pull/181).
+
+## References
+
+- https://bitcointalk.org/index.php?topic=865169.msg16349234#msg16349234
+- https://blog.ethereum.org/2014/11/25/proof-stake-learned-love-weak-subjectivity/
+
+Bespoke previously considered exponential functions:
+
+```go
+/*
+ecbp1100AGExpB is an exponential function with x as a base (and rationalized exponent).
+
+OPTION 2: Slightly slower takeoff, steeper eventual ascent
+g(x)=x^(x*0.00002)
+*/
+func ecbp1100AGExpB(x float64) (antiGravity float64) {
+	return math.Pow(x, x*0.00002)
+}
+```
+
+```go
+/*
+ecbp1100AGExpA is an exponential function with x as exponent.
+
+This was (one of?) Buterin's "original" specs:
+> 1.0001 ** (number of seconds between when S1 was received and when S2 was received)
+- https://bitcointalk.org/index.php?topic=865169.msg16349234#msg16349234
+> gravity(B') = gravity(B) * 0.99 ^ n
+- https://blog.ethereum.org/2014/11/25/proof-stake-learned-love-weak-subjectivity/
+
+OPTION 1 (Original ESS)
+f(x)=1.0001^(x)
+*/
+func ecbp1100AGExpA(x float64) (antiGravity float64) {
+	return math.Pow(1.0001, x)
+}
+```
+
+A further alternative is demonstrated with a _cosin_ function as follows, which, being squared, has an even flatter "ramp up" section. However, since difficulty adjustment is discrete and the
+primary case of consideration in this context is one of equivalent difficulty, efforts toward an ultra-low
+antigravity product seem well-intentioned but practically ineffectual.
+
+```
+s(x)=8 (-cos((0.5 π x)/(6000))+1)^(2)+1
+```
+
+## Copyright/Licensing
+
+Copyright 2020 Isaac A. \<b5c6@protonmail.com\>
```

Comment:
is this copyright for the text of the ecip?

Reply:
Ah, no, this is an accidental vestige of a reused older document. Will remove.

Reply:
LGTM if you could fix the license footer.

Reply:
Same, will approve after it's fixed.

Reply:
https://github.com/ethereumclassic/ECIPs/pull/373/commits/7b06c04b0c5d0ffbbff7bdf4edfa762b750160fa